src/util.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const { isIdentifierStart, isIdentifierChar } = require('acorn');
+const $multiLine = Symbol('IO.kNeedsAnotherLine');
```

Comment:
generally we name symbol constants with `k`, so this would be `kMultiLine`

Reply:
Sure, will look into this.

Reply:
Shouldn't it be [`multiline`](https://www.collinsdictionary.com/us/dictionary/english/multiline) _(lower and not camel which would suggest `multi line`)_.

Reply:
@jdalton Yup. I have to move this around as @devsnek suggested.